mapping.yaml

```diff
@@ -33,7 +33,9 @@ languages:
     - russian_yavert
     - russian_yazhert
   be_BY:
-    - belarusian
+    - belarusian_cyrillic
+    - belarusian_latin
+    - belarusian_latin2
```

Comment:
The latin layouts should be for a separate language, `be_Latn`

Reply:
Done